docs/dscom.build/ReadMe.md

```diff
@@ -0,0 +1,229 @@
+          ╚═══════════════╪══════════════════════════════════╪═════════════════════════════════════════════════════════════════════════════════╪═════════════════════════════╪═══════════════════════════════╪═════════════════════════════╪═════════════════════════════╪═══════════════════════╪════════════════════════════╪════════════════════════════════╪═════════════════════╪═════════════════════╝
+                    ┌─────┴─────┐                       ┌────┴────┐                                                                     ┌──────┴──────┐          ┌───────────┴────────────┐          ┌───────┴────────┐          ┌─────────┴─────────┐          ┌────────┴────────┐          ┌───┴────┐          ┌────────────┴─────────────┐          ┌───────┴────────┐          ┌─┴─┐                    
+                    │BuildEngine│                       │TlbExport│                                                                     │IBuildContext│          │TypeLibConverterSettings│          │FileSystemChecks│          │AssemblyLoadContext│          │AppDomain.Current│          │Assembly│          │LoggingTypeLibExporterSink│          │TypeLibConverter│          │Log│                    
+                    └───────────┘                       └─────────┘                                                                     └─────────────┘          └────────────────────────┘          └────────────────┘          └───────────────────┘          └─────────────────┘          └────────┘          └──────────────────────────┘          └────────────────┘          └───┘                    
+```
+
+## Test
+
+### Unit tests
+
+The class `TlbExport` is written with a level of abstraction and hence is tested with a [unit test](../../src/dscom.test/tests/BuildTaskTest.cs).
+
+### Acceptance tests
+
+The examples [32-bit](../../examples/32bit/) and [outproc](../../examples/outproc/) each provide an `msbuild-acceptance-test.cmd` script. This file builds and packages the build task and adds it to the `.csproj` file of the resulting assembly. Then the build is conducted. It must be warning free.
+
+## Packaging
+
+The packaging will contain the executable file for [dscom CLI](../../src/dscom.client/dscom.client.csproj) for both platforms, x64 and x86. The configuration is Release. The executable will be published in Release as a framework dependent deployment using a [single file](https://learn.microsoft.com/en-us/dotnet/core/deploying/single-file/overview?tabs=cli), which means that [dSPACE.Runtime.InteropServices](../../src/dscom/dscom.csproj) and [System.CommandLine](https://packages.nuget.org/packages/System.CommandLine/) are bundled, but no AOT takes place and no framework files will be part of the bundle.

```

Comment:
One thing I noticed: You have pretty long lines with multiple sentences. It would improve maintainability if you split those long lines into several shorter ones.

Reply:
Yeah, I know, but since I'm using word wrap this is ok for me. 

For me it is harder to maintain if there were a custom line break limit.

What would be your maximum column width that is acceptable?

Reply:
We currently don't have a formal maximum line length but as a rough rule: 
I try to start a new line on every new sentence. Unless it's obvious the next sentence does fit into the current line.  
You don't have to split your lines within a sentence. 

It makes reviews and comparisons much easier in my experience.

Reply:
Done in 5f64d7b75e3dcd7dbbfc0896d5e6a382de4b20dc